prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -395,10 +413,8 @@ def prompt_processor() -> Any:
                     characters. No explanation is required.\
                     If you cannot extract the number, output 0."
                 answer = run_completion(
-                    llm_helper=llm_helper,
-                    llm_li=llm_li,
-                    prompt=prompt,
-                    adapter_instance_id=adapter_instance_id,
+                    llm,
+                    prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
DOne